rubocop.yml

```diff
@@ -13,9 +13,12 @@ Lint/AmbiguousOperator:
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
 Lint/AssignmentInCondition:
-  AllowSafeAssignment: false
+  AllowSafeAssignment: true
```

Comment:
not a fan and it has been discussed long time ago, when it ended up in our conventions. I remember at least one bad experience concerning this and it makes things a bit clearer. :-1: from my side.